webauth/index.js

```diff
@@ -54,7 +54,7 @@ export default class WebAuth {
       }/${bundleIdentifier}/callback`;
       const expectedState = options.state || state;
       let query = {
-        ...defaults
```

Comment:
Was this even working before? Seems like a syntax error:

https://github.com/auth0/react-native-auth0/blob/master/webauth/index.js#L57

Reply:
No, was in a previous previous 3d party PR. This is a fix for that.